tutorials/Threshold_Weighted_Scores.ipynb

```diff
@@ -5,14 +5,23 @@
    "metadata": {},
    "source": [
     "# Threshold weighted scores\n",
+    "\n",
     "## Introduction\n",
-    "For a long time, people have wanted to be able to evaluate the performance of forecasts of extreme events using verification methods that can't be hedged. Hedging occurs when there is a difference between a forecaster's judgement and their forecast [(Murphy 1978)](https://doi.org/10.1175/1520-0477(1978)059<0371:HATMOE>2.0.CO;2). In the context of an automated forecast system, hedging occurs when the forecast produced to optimise the expected verification score differs from a forecast produced that aligns with the prescribed forecast directive (e.g., forecast the 90th percentile wind speed). The threshold weighted scores in `scores` allow us to do this using consistent scoring functions.\n",
+    "For a long time, people have wanted to be able to evaluate the performance of forecasts of extreme events using verification methods that can't be hedged. This bring together two concepts - avoidance of hedging, particularly of probabilities, and also a focus on extreme conditions (e.g. very hot conditions).\n",
+    "\n",
+    "Hedging occurs when a forecast is optimised for a verification metric rather than for a forecast directive (see [Murphy (1978)](https://doi.org/10.1175/1520-0477(1978)059<0371:HATMOE>2.0.CO;2)). In the context of an automated system, hedging occurs when the model or system is designed or trained according to a verification metric which is misaligned to the forecast directive.\n",
+    "\n",
+    "An example of a forecast directive is to predict the 90th percentile wind speed. Evaluating the 90th percentile forecast wind speed against observed wind speeds would needlessly penalise the forecast if scored using mean absolute error (MAE), since that metric optimises for the 50th percentile wind speed. \n",
```

Comment:
```suggestion
    "An example of a forecast directive is to predict the 90th percentile wind speed. If the 90th percentile forecast wind speed forecasts were evaluated against observed wind speeds using mean absolute error (MAE), the forecaster or forecast system developer is faced with the dilemma of whether to produce forecasts that are consistent with the forecast directive or if they should be produced to optimise the expected MAE (since MAE optimises for the 50th percentile wind speed). \n",
```